### ISSUE TITLE
animation until the server return the data

### ISSUE THREAD

**Issue by IsidroMar95** [jamesdordoy/laravel-vue-datatable#47]
it would be great if you could add some extra features, I don't know how complicated it is e.g.:

1. show an animation until the server return the data.

**Comment by jamesdordoy** [comment]
As Vue offers its own transitions & animations i'm sure it wouldn't be too complicated to implement.

I'll have a look at adding something tomorrow.

**Comment by IsidroMar95** [comment]
thanks @jamesdordoy one more question how can i make each row clickable, i mean each row be a link?

**Comment by jamesdordoy** [comment]
See [here](https://jamesdordoy.github.io/laravel-vue-datatable/examples/override-table-body).

You have a complicated data table aha.

**Comment by IsidroMar95** [comment]
Ahaha I know but thank you! You helped me a lot 💪🏾

**Comment by jamesdordoy** [comment]
I've added an example using an overlay animation for when the table is loading [here](https://jamesdordoy.github.io/laravel-vue-datatable/examples/loading-animations).

I tried adding a loading icon directly to the table body but it made body collapse depending on the amount of data that was showing and looked really strange.

In my opinion, this was the nicest way to add an a loading animation but i'm open to other suggestions.
